mobile.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/mobile.css">
```

Comment:
- [ ] I can not see your CSS styles due to you are using a bad practice `href` value, when you are calling a file that is at the same folder as your HTML code. you should use the next value `href="./mobile.css"` instead of `href="/mobile.css"`.